### DISCUSSION TITLE
Dex1 XDrip Oneplus

### DISCUSSION THREAD

**Discussion by TWilhoWI** [NightscoutFoundation/xDrip#3256] [Bug reports]
When my Dexcom One can't connect for a single moment to XDrip it gets unbounded. Phone state service disconnected. I have to restart the collector and wait for a couple of Moments till it resends the connection request which Is manually have to approve. Followed the best practice guidelines for setting up. Got already a couple of transmitters with the same bug. This is especially not Fun at night where a don't seem the error till next morning.

**Comment by Navid200** [comment]
Please post a screenshot of the following xDrip settings page.

Settings &#8722;> G5/G6/Dex1 Debug Settings

**Comment by TWilhoWI** [comment]
Here you go!
![20231227_111107](https://github.com/NightscoutFoundation/xDrip/assets/120023149/606579b5-f52b-4451-8816-49b1c326b457)
![20231227_111042](https://github.com/NightscoutFoundation/xDrip/assets/120023149/2876da95-8a89-42fe-841e-486d579941d6)

**Comment by Navid200** [comment]
What is the phone brand and model?
Can you post a screenshot of the G5/G6/G7 status page?  https://navid200.github.io/xDrip/docs/StatusG5G6.html
Have you disabled battery optimization?

**Comment by TWilhoWI** [comment]
Yes of course, this is the status Page when connected. No screenshot of the non connected state. Phone is Oneplus 8T. Battery optimalisation is disabled.
![20231227_142727](https://github.com/NightscoutFoundation/xDrip/assets/120023149/a9c4bb3a-9107-4b6f-b419-fb1d666de079)

**Reply by Navid200** [reply]
Please enable the following setting and let us know if it makes things better.

Settings &#8722;> G5/G6/Dex1 Debug Settings &#8722;> Special Pairing Workaround

**Reply by TWilhoWI** [reply]
No connections issues so far! Thanks a lot!
